server/libs/defaults.js

```diff
@@ -18,6 +18,11 @@ var defaultSettings = {
    "wlanPassphrase": "deinPW",
    "wlanSSID": "unbekanntes WLAN",
    "motorWarningChecked": false,
-   "addToSensorVal": 0
+   "addToSensorVal": 0,
```

Comment:
@jonathanschneider Does storing and object of objects work? I'm having a hard time to recap, but there might have been a reason that made me keep the settings object flat. Workaround camelCase: `telegramEnabled`, `telegramToken`, `telegramChatId`

Reply:
Defining an object in a schema is possible according to https://mongoosejs.com/docs/guide.html but if the storing in settings.html works I don't know yet. Haven't gotten that far to try to access the object.

Reply:
Yup, it works!